pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -20,6 +20,11 @@ import (
 )
 
 func validateInput(c utils.CommandLine, pluginFolder string) error {
```

Comment:
`pluginFolder` argument isn't used inside this function and could be removed as well

Reply:
you can remove it then :)